mitreattack/navlayers/core/exceptions.py

```diff
@@ -105,3 +105,21 @@ def loadChecker(caller, testee, required, field):
         if entry not in testee:
             handler(caller, '{} is not present in {} [{}]'.format(entry, field, testee))
             raise MissingParameters
+
+
+def handle_object_placement(handle_to_self_field, potential_object, objectType, list=False):
+    """
+        If the input element (potential_object) is an object of objectType, assign it to handle_to_self_field
```

Comment:
The docstring documents behavior but not intention. Might be worth also saying why this should be used, e.g "helper for assigning fields with typechecking" (assuming I'm correct about the intention).

Reply:
This may not actually belong in exceptions if it mutates the state of an object. While it does perform validation, nowhere does it raise and nor is it a class inheriting from Exception.

Reply:
I updated the documentation to be more explicit about what it's doing rather than just describing the effects. I originally had put this in exceptions because I felt a bit silly making an entire new 'helpers' file for a single function, but I've gone ahead and done that in the anticipation that eventually there will be more functions in said file.